src/ffi/spatialmath/quaternion.rs

```diff
@@ -216,10 +222,10 @@ pub unsafe extern "C" fn normalize_quaternion(quat_ptr: *mut Quaternion) {
 /// The caller must remember to free the quaternion memory of 
 /// *both* the input and output quaternions when finished with them 
 /// using the free_quaternion_memory FFI function
-#[no_mangle]
-pub unsafe extern "C" fn quaternion_get_normalized(quat_ptr: *const Quaternion) -> *mut Quaternion {
+// #[no_mangle]
```

Comment:
why are we commenting out the `#[no_mangle]` here? that seems intuitively wrong to me, but even if I'm mistaken then we should just delete it rather than commenting it out.

Reply:
It was a mistake